infrastructure/database.tf

```diff
@@ -100,6 +100,12 @@ resource "aws_iam_policy_attachment" "db_accessor_2" {
   policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM"
 }
 
+resource "aws_iam_policy_attachment" "db_accessor_3" {
+  name       = "crossfeed-db-accessor-${var.stage}"
+  roles      = [aws_iam_role.db_accessor.id]
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonS3FullAccess"
```

Comment:
Should we be giving full S3 access? Or just the specific buckets needed?

Reply:
adjusted in [9e2d203](https://github.com/cisagov/crossfeed/pull/1591/commits/9e2d2035247009fe97390760d950c871b9c5cd26)